froyo-event.md

```diff
@@ -0,0 +1,14 @@
+Title: Froyo Social
+Date: 2016-10-10
+Category: Events
+Tags: social
+Slug: froyo-social-f16
+Author: Niv Govindaraju
+Summary: Come out to talk to some awesome women in tech while bonding over delicious froyo!
+
```

Comment:
Wanna add a description? Otherwise itll look like a real boring post on the website :P